spec/factories/conferences.rb

```diff
@@ -1,7 +1,9 @@
 FactoryGirl.define do
   factory :conference do
     name { [FFaker::CheesyLingo.title, 'Conf'].join ' ' }
-    tickets 2
```

Comment:
I removed the tickets, because they are not a requirement for validation. Plus, the only test that refers to the ticket attributes, overrides it.
(Copied from old PR)